dashboard/state_aggregator.py

```diff
@@ -362,13 +362,15 @@ async def list_tasks(self, *, option: ListApiOptions) -> ListApiResponse:
         """
         job_id = None
         for filter in option.filters:
-            if filter[0] == "job_id":
+            if filter[0] == "job_id" and filter[1] == "=":
```

Comment:
This was a bug.

Reply:
Mind adding a unit test that can fail this?

Reply:
The new unit test I added actually exposes it.